lib/src/widgets/movie_seach_bar.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+
+import '../bloc/movies_bloc.dart';
+import '../utils/constants.dart';
+
+class MovieSearchBar extends StatelessWidget {
+  const MovieSearchBar({
+    Key key,
+    this.bloc,
+  }) : super(key: key);
+
+  final MoviesBloc bloc;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: Constants.appBarTextFieldHeight,
+      child: TextField(
+        textAlignVertical: TextAlignVertical.bottom,
+        decoration: InputDecoration(
+          hintText: Constants.appBarTextFieldHint,
+          filled: Constants.appBarTextFieldIsFilled,
+          fillColor: Colors.blueGrey.shade100,
+          enabledBorder: OutlineInputBorder(
+            borderSide: BorderSide(
+              color: Colors.blueGrey.shade100,
+              width: Constants.appBarTextFieldBorderWidth,
+            ),
+          ),
+          suffixIcon: Icon(
+            Icons.search,
+          ),
+          contentPadding: EdgeInsets.fromLTRB(
+            Constants.appBarTextFieldContentPadding,
+            Constants.appBarTextFieldContentPadding,
+            Constants.appBarTextFieldContentPadding,
+            Constants.appBarTextFieldContentPadding,
```

Comment:
If you use the same constant for LTRB, change it to EdgeInsets.all(Constants.appBarTextFieldContentPadding)

Reply:
Great! Done